Formula/delve.rb

```diff
@@ -1,9 +1,9 @@
 class Delve < Formula
   desc "Debugger for the Go programming language."
   homepage "https://github.com/derekparker/delve"
-  url "https://github.com/derekparker/delve/archive/v0.12.1.tar.gz"
-  version "0.12.1"
-  sha256 "7ca46626b312fcd2873c7bd86149b7bb2a707b0f285cb984728fa6922a24ef26"
+  url "https://github.com/derekparker/delve/archive/v0.12.2.tar.gz"
+  version "0.12.2"
+  sha256 "f6057161606f66f1197c96ad8b6a9580ee1bb68b"
```

Comment:
This isn't a sha256, this is git's sha-1. I'm not sure but I think this needs to be the sha256 of the file.